staging/src/k8s.io/api/batch/v1/register.go

```diff
@@ -46,6 +46,9 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 	scheme.AddKnownTypes(SchemeGroupVersion,
 		&Job{},
 		&JobList{},
+		&JobTemplate{},
```

Comment:
I don't see JobTemplate being used anywhere... can we delete it?

Reply:
Removed.